src/System.Diagnostics.Process/src/System/Diagnostics/Process.Win32.cs

```diff
@@ -400,11 +406,6 @@ private IEnumerable<Exception> KillTree(SafeProcessHandle handle)
                 // This method can return before stopping has completed. Down the road, could possibly wait for termination to complete before continuing.
                 Kill();
             }
-            catch (InvalidOperationException)
-            {
-                // The process isn't in a valid state for termination (e.g. already dead), so ignore the exception
-                // but don't give up in case children can still be enumerated.
-            }
```

Comment:
This exception is no longer possible in any circumstance?

Reply:
If it does, I think it means something is broken with the implementation.

Reply:
Ok.  Let's make sure we're confident in our tests here then :wink:

Reply:
I looked closer at the implementation. `Kill` will only throw `InvalidOperationException` if the process is not associated or not local. All processes here will be associated & local, so no `InvalidOperationExceptions` are expected.

Reply:
Thanks for checking!